src/icon-manifest/manifestMerger.ts

```diff
@@ -1,49 +1,39 @@
 import * as _ from 'lodash';
-import {
-  IFileCollection,
-  IFolderCollection,
-  IExtensionCollection,
-  IExtension,
-  FileFormat,
-  IIconSchema,
-  IIconGenerator,
-  IFileDefault,
-  IFolderDefault,
-} from '../models';
+import * as model from '../models';
```

Comment:
The benefit is fewer lines in files, something that `bithound` is complaining about for many of the project's files.

I don't agree with your claim that we lose readability. Now when I see `IFolderDefault`, for example, I know it can be found in `models`. Formerly, I had to scroll back up to the `import` to figure it out.